### PR TITLE
Slight improvements to the README

### PR DESCRIPTION
**Pull request by JohnMcLear**
* Replace "this" where appropriate with Lightspeed providing accurate context.
* Generally improve language and readability
* Minor formatting changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,16 +173,18 @@ npm build
...
+The above will serve the build folder on port 80.
+
+View Lightspeed in your web browser by visiting http://hostname or http://your.ip.address.here
 
 
 <!-- _For more examples, please refer to the [Documentation](https://example.com)_ -->
 
 ## Streaming From OBS
 
-By default since we are using the FTL protocol you cannot just use a custom server. You will need to edit your `services.json` file. It can be found at `%AppData%\obs-studio\plugin_config\rtmp-services\services.json` on Windows and `/Users/YOURUSERNAME/Library/Application\ Support/obs-studio/plugin_config/rtmp-services/services.json`
```
Comment (GRVYDEV) [inline README.md:183]: This is backwards. The %AppData% one is Windows and the /Users is Mac

**Review by GRVYDEV — COMMENTED** [review]
Looks great! Thank you! Just fix the part involving where the services.json file is (First path is windows and second path is macOS) and it'll be ready to merge!

File: README.md (hunk 1 above)
Comment (JohnMcLear) [reply README.md:183]: Good spot.

**Review by GRVYDEV — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**